doc/contributing/adding-v8-fast-api.md

```diff
@@ -38,6 +38,23 @@ for example, they may not trigger garbage collection.
 * The fast callback must be idempotent up to the point where error and fallback
   conditions are checked, because otherwise executing the slow callback might
   produce visible side effects twice.
+* If the receiver is used in the callback it must be passed as a second argument
+  leaving the first one unused, to prevent the JS land from accidentally omitting the receiver when
+  invoking the fast API method.
```

Comment:
```suggestion
* If the receiver is used in the callback, it must be passed as a second argument,
  leaving the first one unused, to prevent the JS land from accidentally omitting the receiver when
  invoking the fast API method.
```